docs/Installing.asciidoc

```diff
@@ -774,7 +774,7 @@ automatically every day. It is also possible to trigger the cleanup manually by
 ** needle_delete needle_modify
 
 Some of these events are very common and may clutter audit database. For this reason `job_grab` and `job_done`
-events are blacklisted by default.
+events are blocklisted by default.
```

Comment:
please avoid this verb

Reply:
Is blacklisted really better than blocklisted? Should we write "For this reason `job_grab` and `job_done` are on the blocklist by default"?

Reply:
ok, did "on the blocklist by default" now